docs/developers/protocol/hermez-protocol/protocol.md

```diff
@@ -611,15 +609,15 @@ Sender sends the transaction to a Ethereum address recipient in the state tree.
 If the recipient does not exist and coordinator wants to process the transaction, it should create a new account with the recipient Ethereum address. 
 
 It is assumed that the `toIdx` is set to the special index 0.
-`toEthAddr` would be used to choose a recipient to transfer the `amountFloat16`. 
-Hence, coordinator would select the recipient `idx` to add `amountFloat16` (called `auxToIdx`).
+`toEthAddr` would be used to choose a recipient to transfer the `amountFloat40`. 
+Hence, coordinator would select the recipient `idx` to add `amountFloat40` (called `auxToIdx`).
 
 > Note that this transaction encourages the coordinator to create new accounts through L1 coordinator transaction [CreateAccountEth](#createaccounteth).
 > It is important to mention that this kind on transactions allows the creation of new accounts in the state tree without the need of having any `ether` on L1. Hence, users could create new accounts and deposit tokens just through an L2 transaction. 
```

Comment:
"It is important to mention that these kinds of transactions allow the creation of new accounts...."